chill-scala/src/main/scala-2.12-/com/twitter/chill/ScalaCollectionsRegistrarCompat.scala

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2019 Twitter, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.twitter.chill
+
+class ScalaCollectionsRegistrarCompat extends IKryoRegistrar {
```

Comment:
Yeah. This is here to avoid doing scala version checks to add the
`newK.register(classOf[Range.Exclusive])` for `2.13` Maybe we should add this as comment?

Reply:
Yeah I think explaining this for the person working on 2.14 who is confused it would be useful.